pkg/apis/nsx.vmware.com/v1alpha1/staticroute_types.go

```diff
@@ -31,7 +31,8 @@ type NextHop struct {
 
 // StaticRouteStatus defines the observed state of StaticRoute.
 type StaticRouteStatus struct {
-	Conditions []StaticRouteCondition `json:"conditions"`
+	Conditions      []StaticRouteCondition `json:"conditions"`
+	NSXResourcePath string                 `json:"nsxResourcePath"`
```

Comment:
I remember we removed nsxResourcePath

Reply:
ok

Reply:
ditto

Reply:
why we still have nsxResourcePath here?

Reply:
waiting for another PR, could ignore this.